packages/flutter_tools/test/src/context.dart

```diff
@@ -82,7 +82,7 @@ void testUsingContext(
             when(mock.getAttachedDevices()).thenAnswer((Invocation _) async => <IOSSimulator>[]);
             return mock;
           },
-          OutputPreferences: () => OutputPreferences(showColor: false),
+          OutputPreferences: () => OutputPreferences(showColor: false, wrapText: false),
```

Comment:
Unfortunately, we have two ways of creating a context for testing. The other one is in testbed.dart, and should probably get a similar update.

Reply:
luckily it's in the OKRs :)